docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,367 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-regression]]
+=== Predicting delays with {reganalysis} by using Kibana sample flight data
+
+
+The goal of <<dfa-regression,{reganalysis}>> is to find relationships among 
+different features of your data, then make predictions on the new data points 
+based on the previously found relationships. 
+
+Let's try to predict flight delays by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We use this data to learn 
+a model that allows us to predict flight delays based on information such as 
+weather and location of the destination and origin, flight distance and carrier. 
+We'd like to predict the number of minutes delayed for each flight. As it is a 
+continuous numeric variable, we'll use {reganalysis} to make the prediction.
+
+This dataset consists of sample data that has been manually created and it 
+contains inconsistencies. For example, a flight can be both delayed and 
+canceled. However, it is easy to set up in Kibana and the use case is 
+applicable, so it is suitable to be an example. Still, don't forget this is only 
+sample data, do not expect adequate results.
```

Comment:
For added insight, suggest

`Despite the data inconsistencies, flight delays can be used to demonstrate regression analysis because
it is easy to set up in Kibana, the use case is relevant and it is a good example of how the 
quality of input data will affect the quality of results.`